LICENSE

```diff
@@ -284,6 +284,22 @@
 
     3rdparty/openmp
 
+    =======================================================================================
+    Apache-2.0 license + Microsoft Copyright
+    =======================================================================================
```

Comment:
The License file should not contain information about copyright. Copyright notices go to NOTICE file but only when legally required: https://infra.apache.org/licensing-howto.html#mod-notice